runtime/src/main/java/org/corfudb/protocols/wireprotocol/TokenRequest.java

```diff
@@ -88,6 +92,7 @@ public TokenRequest(ByteBuf buf) {
 
         switch (reqType) {
 
+            case QUERY_ALL_STREAMS:
```

Comment:
It appears that there is a deserialization bug that causes this PR to break network requests.

```
 01:35:48.406 ERROR [worker-3] o.c.i.NettyServerRouter - Error in handling inbound message, {}
 io.netty.handler.codec.DecoderException: java.lang.RuntimeException: Attempt to deserialize a message which is not a CorfuMsg, Marker = 0 but expected 0xC0FC0FC0
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
```

This appears to be because you serialize the number of numTokens, but fail to deserialize it, causing a frame synchronization error.

Also, please follow the naming convention and prefix ```QUERY_ALL_STREAMS``` with ```TK_```. Perhaps ```TK_QUERY_ALL``` would be better.